game.rb

```diff
@@ -3,7 +3,7 @@
 class Game < Item
   attr_accessor :multiplayer, :last_played_at
 
-  def initialize(multiplayer, last_played_at, publish_date)
+  def initialize(multiplayer, _last_played_at, _publish_date)
     super(date)
     @multiplayer = multiplayer
     @last_played_at = date
```

Comment:
Please make sure this matches the initialized variable (use either last_played_at or date). We also need to parse the string input to turn it into a date. For example:
Date.parse(last_played_at) or Date.parse(date)